src/components/molecules/ReactionsBar/ReactionsBar.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback } from "react";
+import { faVolumeMute, faVolumeUp } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { addReaction } from "store/actions/Reactions";
+
+import { EmojiReactionType, ReactionData } from "types/reactions";
+
+import { createEmojiReaction } from "utils/reactions";
+
+import { useDispatch } from "hooks/useDispatch";
+import { useUser } from "hooks/useUser";
+
+import { Reaction } from "components/atoms/Reaction";
+
+import "./ReactionsBar.scss";
+
+export interface ReactionsBarProps {
+  reactions: ReactionData<EmojiReactionType>[];
+  venueId: string;
+  onClickLeaveSeat?: () => void;
+  isAudioEffectDisabled: boolean;
+  onClickMute: () => void;
+}
+
+export const ReactionsBar: React.FC<ReactionsBarProps> = ({
+  reactions,
+  venueId,
+  onClickLeaveSeat,
+  isAudioEffectDisabled,
+  onClickMute,
```

Comment:
muteReactions

Reply:
Possibly also rename `isAudioEffectDisabled` at the same time if you think it makes sense to? I assume they both relate to the same concept? If so, they should be aligned IMO.

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1339/commits/26538259b006366b9db20520d15cf6def9cea65e